db/migrate/20160425161234_remove_deleted_tables_from_replication_settings.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveDeletedTablesFromReplicationSettings < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    deleted_tables = %w(miq_events miq_license_contents vim_performances)
```

Comment:
The table was renamed [here](https://github.com/ManageIQ/manageiq/pull/1527/commits/651469e1d230302befe9ec39b4aa39cf887c6148)  in https://github.com/ManageIQ/manageiq/pull/1527, but it was never removed from the exclude_table setting.